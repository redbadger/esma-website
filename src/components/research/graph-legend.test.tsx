import React from "react";
import renderer from "react-test-renderer";
import GraphLegend, { LegendVariant } from "./graph-legend";

describe("GraphLegend", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <GraphLegend
          variant={LegendVariant.PILL}
          labels={[{ color: "aqua", label: "Test" }]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
