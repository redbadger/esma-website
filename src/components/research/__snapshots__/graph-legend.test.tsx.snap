// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GraphLegend renders correctly 1`] = `
<div
  className="css-1mjgarr-graphLegendStyles"
>
  <div
    className="legend-wrapper"
  >
    <div
      className="css-1lwf0ie-legendMarkerStyles"
    />
    <span>
      Test
    </span>
  </div>
</div>
`;
